src/Header/Header.js

```diff
@@ -13,27 +13,27 @@ const Header = ({ loginStatus, logOut, name }) => {
         <Link to="/"><h1>Rancid Tomatillos</h1></Link>
       </div>
       {loginStatus && (
-        <div className="logged-in">
-          <h2 className="username">Hello, {name}</h2>
-          <div className="logout-button">
-            <img
-              src={clapboard}
-              onClick={logOut}
-              alt="Movie-Production Clapboard"
-            />
-            <p onClick={logOut}>Log Out</p>
-          </div>
+      <div className="logged-in">
```

Comment:
I agree that this is an appropriate use of divs as all they are doing is dividing these elements for styling purposes.